src/test/java/uk/gov/hmcts/reform/civil/validation/PartyValidatorTest.java

```diff
@@ -0,0 +1,167 @@
+                .county("Proper address")
+                .postCode("Proper address")
+                .postTown("Proper address")
+                .build();
+
+            List<String> errors = new ArrayList<>();
+            validator.validateAddress(primaryAddress, errors);
+
+            assertThat(errors).isEmpty();
+        }
+
+        @Test
+        void should_not_return_Special_character_error_address_line1() {
+            Address primaryAddress = Address.builder()
+                .addressLine1("ˆProper address")
+                .build();
+
+            List<String> errors = new ArrayList<>();
+            validator.validateAddress(primaryAddress, errors);
+            assertThat(errors).isNotEmpty();
+        }
+
+        @Test
+        void should_not_return_Special_character_error_address_line2() {
+            Address primaryAddress = Address.builder()
+                .addressLine1("Proper address")
+                .addressLine2("`Proper address")
+                .build();
+
+            List<String> errors = new ArrayList<>();
+            validator.validateAddress(primaryAddress, errors);
+            assertThat(errors).isNotEmpty();
+        }
+
+        @Test
+        void should_not_return_Special_character_error_address_line3() {
+            Address primaryAddress = Address.builder()
+                .addressLine1("Proper address")
+                .addressLine3("´Proper address")
+                .build();
+
+            List<String> errors = new ArrayList<>();
+            validator.validateAddress(primaryAddress, errors);
+            assertThat(errors).isNotEmpty();
+        }
+
+        @Test
+        void should_not_return_Special_character_error_address_posttown() {
+            Address primaryAddress = Address.builder()
+                .addressLine1("Proper address")
+                .postTown(" ¨Proper address")
+                .build();
+
+            List<String> errors = new ArrayList<>();
+            validator.validateAddress(primaryAddress, errors);
+            assertThat(errors).isNotEmpty();
+        }
+
+        @Test
+        void should_not_return_Special_character_error_address_postcode() {
+            Address primaryAddress = Address.builder()
+                .addressLine1("Proper address")
+                .postCode(" ¨Proper address")
+                .build();
+
+            List<String> errors = new ArrayList<>();
+            validator.validateAddress(primaryAddress, errors);
+            assertThat(errors).isNotEmpty();
+        }
+
+        @Test
+        void should_not_return_Special_character_error_address_county() {
+            Address primaryAddress = Address.builder()
+                .addressLine1("Proper address")
+                .county(" ¨Proper address")
+                .build();
+
+            List<String> errors = new ArrayList<>();
+            validator.validateAddress(primaryAddress, errors);
+            assertThat(errors).isNotEmpty();
```

Comment:
Shouldn't the assertions contain the expected messages, rather than just notEmpty?